README.md

```diff
@@ -104,22 +109,117 @@ Disable Algolia Search custom JS file that initializes the Search Box and Hits w
 add_filter('algolia_integration_disable_instant_search_custom_js', '__return_true' );
 ```
 
-## Shortcodes
-### Search Box
+## Instant Search widgets
+The plugin comes with 2 Instant Search widgets preconfigured: [Search Box](https://community.algolia.com/instantsearch.js/v2/widgets/searchBox.html) and [Hits](https://community.algolia.com/instantsearch.js/v2/widgets/hits.html) widgets.
+
+This widgets will search by default on all post types that you marked for syncing in the Plugin's Settings.
+
+### Search Box widget
 Print the Instant Search Box widget using this shortcode:
 
 `print_algolia_search_box`
 
+##Hits widget (Results)
+
 Print the Instant Search Box hits (results) widget using this shortcode:
 
 `print_algolia_results`
 
-## Change the hits item template
+### Customize the widgets settings
+
+## Set Post Types
+
+By default, the widgets will search on the post types selected on the Setting's page.
+
+To change the post types used by the widgets, use the following filter:
+
+Example:
+```php
+add_filter( 
+    'algolia_integration_widgets_post_types', 
+    function() { 
+        return [
+                'post',
+                'custom_post_type_1',
+                'custom_post_type_2',
+            ],
+        ];
+    } 
+);
+
+Note: Make sure the post types added are selected in the Plugin's Settings page or the widgets won't initialize. 
```

Comment:
Why don't we have a sensible default of say all post types selected except the unlikely ones like attachment?